apps/guide/settings/dev.py

```diff
@@ -1,7 +1,7 @@
 from .base import *  # noqa
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
question: why not? this is the dev settings file

Reply:
I have literally 0 recollection of changing that :sweat_smile: